### ISSUE TITLE
Unable to use Zoom via SSO authentication

### ISSUE THREAD

**Issue by Choens** [flathub/us.zoom.Zoom#431]
I tried to use Zoom today and was unable to authenticate via SSO because Zoom (now) requires a minimum of 5.16.10 (668) for this function. I am using 5.16.6.382, which is the most recent version available on Flathub. The version available for download on zoom.us/download for Fedora Linux is Version 5.17.1 (1840). I don't see any way of getting a history so I'm not sure how out of date the .16 series is.

This is significant because I was actually logged in as a user and successfully used it a couple of weeks ago, but today was forcibly logged out of my account.

I was able to attend the session, but I was not able to host it as intended.

Screenshot:
![image](https://github.com/flathub/us.zoom.Zoom/assets/424993/b66a0f30-275b-4d6e-9649-e794494c4eab)

**Comment by Choens** [comment]
@takluyver I saw your comment on #429 about not knowing when the old Zoom is going to stop working. I would say that process has started. The only account type I can check against is SSO, so I don't know if this is impacting other account types, but that process has now started.

**Comment by takluyver** [comment]
I think we can upgrade to 5.16.10.668 OK; I've just opened #433 for that. But if we can't figure out how to make newer versions work (see #429), I imagine you'll hit the same thing again in a matter of weeks or months.

For my non-SSO account, I can still sign in with 5.16.6 at the moment.

> I don't see any way of getting a history so I'm not sure how out of date the .16 series is.

There are release notes here, which say when each version was released: https://support.zoom.com/hc/en/article?id=zm_kb&sysparm_article=KB0068973

5.16.6 is from October, 5.16.10 from November.
